openjfx/LICENSE

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2015, 2018, Oracle, OpenJFX and/or its affiliates.
```

Comment:
This isn't quite right. For one thing `Oracle and/or its affiliates` is the entity (so should not be split). Also, if there is more than one copyrighting organization, you will want to indicate that some files are Copyright `this organization` and other files are Copyright `that organization` (either with an additional section in the LICENSE file or a separate third-party license file).